scripts/gha/test_simulator.py

```diff
@@ -698,8 +742,8 @@ def _run_instrumented_test():
     "-w", "%s.test/androidx.test.runner.AndroidJUnitRunner" % CONSTANTS[FLAGS.test_type]["android_package"]] 
   logging.info("Running game-loop test: %s", " ".join(args))
   result = subprocess.run(args=args, capture_output=True, text=True, check=False) 
-  if "FAILURES!!!" in result.stdout:
-    _reset_emulator_on_error(_RESET_TYPE_REBOOT)
+  # if "FAILURES!!!" in result.stdout:
```

Comment:
Here, too.

Reply:
Let's keep the code here. The reset func is desired but not working, I will have a separate PR fixing this issue.